### DISCUSSION TITLE
Disambiguate between recurring events with the same name and recurrence pattern

### DISCUSSION THREAD

**Discussion by hittjwX** [davish/obsidian-full-calendar#407] [Feature Requests]
When a user creates a recurring full-day event, `Event Name` which starts 2022-12-01 to 2022-12-06, then a second with the same `Event Name` 2022-12-15 to 2022-12-29, only file `(Every S,F,R,W,T,M,U) Event Name)` gets created. Only the first entry shows on the calendar.

No warning was given for duplicate event names. The expected behavior is two entries, one each per instance.

A user doing time blocking may want to map out `Vacation`, `Travel`, or other generic activities before a proper title can be given. This may also be an event that skips a week.

A workaround is to rename the first recurring full-day event to include the start date in the title. `(Every S,F,R,W,T,M,U) 2022-12-01 Event Name)` and for consistency the second `(Every S,F,R,W,T,M,U) 2022-12-15 Event Name)` for separate notes.

This would work if the note title automatically placed `date: ` in the title consistent with Events and Tasks.

Unique individual notes let users include an agenda, notes, or other relevant details in the note card.

**Comment by davish** [comment]
This is related to #12, but I'll keep this issue open because it gives a nice solution for recurring events.
